src/js/components/Tabs/Tab.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import { compose } from 'recompose';
+
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+
+import { withTheme } from '../hocs';
+
+class Tab extends Component {
+  state = {
+    hover: undefined,
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.active !== nextProps.active && nextProps.active) {
+      this.setState({ hover: undefined });
+    }
+  }
+
+  onClickTab = (event) => {
+    const { onRequestForActive } = this.props;
+    if (event) {
+      event.preventDefault();
+    }
+    onRequestForActive();
+  }
+
+  render() {
+    const { active, title, onMouseOver, onMouseOut, ...rest } = this.props;
+    const { hover } = this.state;
+    delete rest.onRequestForActive;
+
+    const activeTitle = typeof title === 'string' ? (
+      <Text><strong>{title}</strong></Text>
+    ) : title;
+
+    const inactiveTitle = typeof title === 'string' ? (
+      <Text color='dark-4'>{title}</Text>
+    ) : title;
+
+    return (
+      <Button
+        plain={true}
+        role='tab'
+        aria-selected={active}
+        aria-expanded={active}
+        onClick={this.onClickTab}
+        onMouseOver={(...args) => {
+          if (!active) {
+            this.setState({ hover: 'border' });
```

Comment:
Maybe `hover: true`, since I don't think the 'border' is used?

Reply:
this is used here:

```javascript
{ side: 'bottom', size: 'medium', color: hover || 'black' }
```

I believe it is just easier if we keep it `border` instead of having to dealing with boolean to `border` conversion.

Thoughts?